### PR TITLE
Fix the time tests properly

### PR DESCRIPTION
**Pull request by WilliamJamieson**
These the changes I suggested in
https://github.com/astropy/astropy/pull/15646/files#r1404478221

### CODE REVIEW

File: astropy/visualization/tests/test_time.py [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
-import matplotlib.dates
-import matplotlib.pyplot as plt
+import pytest
 from erfa import ErfaWarning
 
 from astropy.time import Time
 from astropy.visualization.time import time_support
 
+matplotlib = pytest.importorskip("matplotlib")
+plt = matplotlib.pyplot
```
Comment (eerovaher) [inline astropy/visualization/tests/test_time.py]: If the `MPL_EPOCH_1970` flag is removed then there is no need to access `matplotlib.dates` anymore, which means these two lines can be combined into 
```suggestion
plt = pytest.importorskip("matplotlib.pyplot")
```

Comment (WilliamJamieson) [reply astropy/visualization/tests/test_time.py]: That is outside the scope of the astropy/astropy#15646

Comment (eerovaher) [reply astropy/visualization/tests/test_time.py]: Removing `MPL_EPOCH_1970` is definitely outside the scope of astropy/astropy#15646, which is why I've opened a separate pull request for that. And although the difference between our suggestions is very small, the cost of rebasing a ~10 line pull request is also very small, so if we can shorten the code by an extra line without code churn then we should.

**Comment by eerovaher** [comment]
Now that astropy/astropy#15646 has been merged, this pull request has become obsolete. Thanks for having had a thorough look at `astropy/visualization/tests/test_time.py` during the review.

**Pull request closed without merging**